src/mechanics/elements/ContinuumElementIGA.cpp

```diff
@@ -55,7 +56,8 @@ const Eigen::VectorXd NuTo::ContinuumElementIGA<TDim>::GetIntegrationPointVolume
     Eigen::VectorXd volume(this->GetNumIntegrationPoints());
     for (int theIP = 0; theIP < this->GetNumIntegrationPoints(); theIP++)
     {
-        Eigen::MatrixXd derivativeShapeFunctionsNatural = this->mInterpolationType->Get(Node::eDof::COORDINATES).CalculateDerivativeShapeFunctionsNatural(theIP, mKnotIDs);
+        const auto ipCoords = this->GetIntegrationType().GetLocalIntegrationPointCoordinates(theIP);
```

Comment:
The problem is that `ContinuumElement` is a base class template, where spezializations don't need to have the same interface. Therefore, the compiler doesn't look for the inherited names. Adding `this->Whatever()` promises to the compiler that all spezializations will have `Whatever()`. The same could be achieved with `using ElementBase::Whatever;` in the `ContinuumElementIGA` class.